bot/exts/evergreen/githubinfo.py

```diff
@@ -0,0 +1,88 @@
+import logging
+from datetime import datetime
+
+import aiohttp
+import discord
+from discord.ext import commands
+from discord.ext.commands.cooldowns import BucketType
+
+log = logging.getLogger(__name__)
+
+
+class GithubInfo(commands.Cog):
+    """Fetches GitHub."""
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    async def fetch_data(self, url: str) -> dict:
+        """Retrieve data as a dictionary."""
+        async with aiohttp.ClientSession() as cs:
+            async with cs.get(url) as r:
+                return await r.json()
+
+    @commands.command(name='github', aliases=['gh'])
+    @commands.cooldown(1, 5, BucketType.user)
+    async def get_github_info(self, ctx: commands.Context, username: str = 'python-discord') -> None:
+        """
+        Fetches a user's GitHub information.
+
+        User defaults to python-discord.
```

Comment:
Maybe it would be better to send help about the command, rather than using python-discord as the default.